src/services/railgun/history/__tests__/transaction-history.test.ts

```diff
@@ -24,6 +35,147 @@ const { expect } = chai;
 const POLYGON_CHAIN: Chain = { type: ChainType.EVM, id: 137 };
 let wallet: RailgunWallet;
 
+const MOCKED_TRANSFER_SEND_TRX: TransactionHistoryItem = {
+  txid: '0x2cedf31ad89e317dab2bc522333c58e9d644e4977c9c7249ca99e3846eb5d652',
+  blockNumber: 100,
+  transferERC20Amounts: [
+    {
+      tokenAddress: '0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270',
+      amountString: '0x435532c61b0800',
+      recipientAddress:
+        '0zk1qygxrr7l92f6wltkpqlqcqstftfn0cn0x5ckyl5tjz6a4kyxnwy9arv7j6fe3z53llg8qt4s7axfdazyrrps78np9pylk4055gkz9e2gd8ulmk0urqt55y3m07t',
+      walletSource: 'railway web',
+    },
+    {
+      tokenAddress: '0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270',
+      amountString: '0x01626218b4586000',
+      recipientAddress:
+        '0zk1qygxrr7l92f6wltkpqlqcqstftfn0cn0x5ckyl5tjz6a4kyxnwy9arv7j6fe3z53llg8qt4s7axfdazyrrps78np9pylk4055gkz9e2gd8ulmk0urqt55y3m07t',
+      walletSource: 'railway web',
+    },
+    {
+      tokenAddress: '0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270',
+      amountString: '0x0120d3a540a09800',
+      recipientAddress:
+        '0zk1qygxrr7l92f6wltkpqlqcqstftfn0cn0x5ckyl5tjz6a4kyxnwy9arv7j6fe3z53llg8qt4s7axfdazyrrps78np9pylk4055gkz9e2gd8ulmk0urqt55y3m07t',
+      walletSource: 'railway web',
+    },
+  ] as RailgunSendERC20Amount[],
```

Comment:
I would suggest declaring this type above, like:

const transferAmounts: RailgunSendERC20Amount[] = ...

This will provide type errors if the type is incorrect.

"as" won't give you an error when the underlying types change - but this can be nice to know that you need to update the tests